ironfish/src/evm/evm.ts

```diff
@@ -33,17 +35,42 @@ export class IronfishEvm {
     const result = await this.runTx(opts)
 
     // TODO(jwp) from custom opcodes populate shields and unshields
+
```

Comment:
Can we add a test for shields here? One where shield comes from non-globaladdress and one that does? We will probably need to define a global address for this.